src/Exception/ObjectGeneratorPersisterException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Fidry\AliceDataFixtures package.
+ *
+ * (c) Théo FIDRY <theo.fidry@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Fidry\AliceDataFixtures\Exception;
+
+class ObjectGeneratorPersisterException extends \Exception
```

Comment:
I would make this class `final` and add a `@private` tag to its phpdoc block as it would be for internal usage only

Reply:
I would also extend `\LogicException` and important the class with use statement like any class. The reasoning for `\LogicException` is that it's not a graceful error one can recover from: it's a config error that requires manual change from the user